src/psyclone/tests/psyclone_test_utils.py

```diff
@@ -109,29 +113,42 @@ class Compile(object):
     :type tmpdir: :py:class:`LocalPath`
     '''
 
-    # Whether or not we run tests requiring code compilation is picked-up
-    # from a command-line flag. (This is set-up in conftest.py.)
-    # pylint: disable=no-member
-    TEST_COMPILE = pytest.config.getoption("--compile")
-    # pylint: enable=no-member
-    # The following allows us to mark a test with @utils.COMPILE if it is
-    # only to be run when the --compile option is passed to py.test
-    COMPILE = pytest.mark.skipif(not TEST_COMPILE,
-                                 reason="Need --compile option to run")
-
-    # pylint: disable=no-member
-    TEST_COMPILE_OPENCL = pytest.config.getoption("--compileopencl")
-    # pylint: enable=no-member
-    COMPILE_OPENCL = pytest.mark.skipif(not TEST_COMPILE_OPENCL,
-                                        reason="Need --compileopencl "
-                                               "option to run")
+    # Class variable to store if compilation is enabled (--compile).
+    TEST_COMPILE = False
+
+    # Class variable to store if opencl compilation is enabled
+    # (--compileopencl).
+    TEST_COMPILE_OPENCL = False
+
+    # Class variable to store the chosen Fortran compiler (--f90, default
+    # gfortran).
+    F90 = "gfortran"
+
+    # Class variable to store the choosen f90 compiler flags (--f90flags).
+    F90FLAGS = ""
+
+    @staticmethod
+    def store_compilation_flags(config):
+        '''This function is called from the conftest session fixture
```

Comment:
Should this say "...session fixture infra_compile" or something?

Reply:
Yes indeed, forgot to update that comment when I had to combine the two functions. Done